thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -0,0 +1,74 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+export default Ember.Component.extend({
+  // TODO implement anomaly feedback
+
+  entities: null, // {}
+
+  anomalyUrn: null, // ""
+
+  anomaly: Ember.computed(
+    'entities',
+    'anomalyUrn',
+    function () {
+      const { entities, anomalyUrn } = this.getProperties('entities', 'anomalyUrn');
+
+      if (!anomalyUrn || !entities || !entities[anomalyUrn]) { return false; }
+
+      return entities[anomalyUrn];
+    }
+  ),
+
+  functionName: Ember.computed('anomaly', function () {
```

Comment:
can a portion of the anomaly change instead? if yes, we'll have to add 'anomaly.attributes' to your computed dependencies. applies to the cp below as well

Reply:
no, the entire anomaly is replaced (see CP above)